index.d.ts

```diff
@@ -111,6 +111,13 @@ declare namespace electronDl {
 		@default false
 		*/
 		readonly overwrite?: boolean;
+
+		/**
+		Save dialog options used to personalize the dialog.
+
+		@defalt {}
+		 */
+		readonly dialogOptions?: any;
```

Comment:
This needs to be strongly typed. Not `any`.

Also needs to be documented in the readme.